Form-Controls/styles.css

```
@@ -0,0 +1,83 @@
+/* General */
+
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* main */
+
+main {
+  font-size: 20px;
+  margin: 3rem 20rem;
+  padding: 1.5rem;
+  height: 100%;
+  line-height: 2rem;
```

```suggestion
  line-height: 2;
```

Never use length units on a line height. It will mess you up in more complex designs.